### PR TITLE
PROJECT STRUCTURE

### PR DESCRIPTION
**Pull request by Amedzro-Elikplim**
# MINIMALIST APP 

- Display to do in index.html
- Bundle javascript using webpack
- Style todo list using flex layout
- Implement modular architecture
- Dynamically iterate over todo array and display each todo list 
- Add input field to accept todo from user

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,28 @@
...
+    description: 'Washing the dishes',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Complete To Do list project',
+    completed: false,
+    index: 1,
+  },
+];
```
Comment (brytebee) [inline src/index.js:15]: - [ ] **Required:** You have done an amazing job so far. However, your list should be loaded in the order of index.

**Hint:** `Array.sort((a, b) => a.index - b.index);`

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Amedzro-Elikplim 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- good documentation✔️
- webpack properly setup✔️
- no linters✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShoiraTa** [comment]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @Amedzro-Elikplim , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — COMMENTED** [review]
**STATUS: REQUIRED CHANGES ✔🍾🎉**
Hello @Amedzro-Elikplim 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- You asked for the changes in a polite and friendly way.
- You documented your work professionally.✔
- Github inline comments feature is used properly.

## Required Changes ♻️
- Since you are repeating the module instead of submitting the old pull request you have to create a new pull request with a new change so that it can be reviewed.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Amedzro-Elikplim** [comment]
Am just doing as told by student success. I already have my project approved and there is no change to make. Why do I need to make a change? In addition, I submitted an old pull request yesterday and it was approved without any problem. Can you please explain your suggestions in detail?

**Comment by emmanuellekamwa** [comment]
You just need to make a small change like adding something to your readme
file and then committing that change. Even I had this type of issue and I
was told to submit a new pull request with a new change so that I can get
my project approved.

**Pull request merged**